src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java

```diff
@@ -80,10 +72,7 @@ public void requestAndWriteKitRequests(List<LatestKitRequest> latestKitRequests)
                                         if (kitType != null) {
                                             KitRequestSettings kitRequestSettings = kitRequestSettingsMap.get(kitType.getKitTypeId());
 
-                                            ArrayList<KitRequest> orderKit = kitsToOrder.get(kitRequestSettings);
-                                            if (orderKit == null) {
-                                                orderKit = new ArrayList<>();
-                                            }
+                                            ArrayList<KitRequest> orderKit = new ArrayList<>();
```

Comment:
could be removed as well =)

Reply:
![Screen Shot 2021-03-17 at 11 40 39 AM](https://user-images.githubusercontent.com/1653048/111495341-a6f3f180-8715-11eb-895c-7bec0e457388.png)

Reply:
How far should this go?

Reply:
![Screen Shot 2021-03-17 at 11 42 45 AM](https://user-images.githubusercontent.com/1653048/111495590-eb7f8d00-8715-11eb-9664-edc74b15ea67.png)

Reply:
Initially I went a bit further--I think my mistake was also removing the call to `addSubKits()`?

Reply:
Done!  One more look?